### PR TITLE
Fix debuggers to get latest  status on apex language server

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Fixes apex debugger and replay debugger not getting the latest status of the Apex Language Server.

### What issues does this PR fix or reference?
#1289

### CODE REVIEW

File: packages/salesforcedx-vscode-apex/src/index.ts [hunk 1]
```diff
@@ -121,7 +121,7 @@ export async function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(commands);
 
   context.subscriptions.push(await registerTestView(testOutlineProvider));
-  const languageClientStatus = languageClientUtils.getStatus();
+  const languageClientStatus = languageClientUtils;
```
Comment (lcampos) [inline packages/salesforcedx-vscode-apex/src/index.ts]: Publishing `languageClientUtils.getStatus()` would not allow the consumers of this api to get the latest server status. Instead, they would get the status of the server at the point in which the apex extension got activated. Most of the time the status would be `Indexing`.

Comment (brpowell) [reply packages/salesforcedx-vscode-apex/src/index.ts]: I'd either change the name of the export from languageClientStatus to languageClientUtils, or export just getStatus with `const languageClientStatus = languageClientUtil.getStatus`.

Comment (AnanyaJha) [reply packages/salesforcedx-vscode-apex/src/index.ts]: So `getStatus()` is instead called by whatever is consuming `languageClientStatus`? ie. `debugConfigurationProvider.ts` to get the status at that point of time?

Comment (vazexqi) [reply packages/salesforcedx-vscode-apex/src/index.ts]: [nitpick] I agree with Bryan on the name.

Comment (lcampos) [reply packages/salesforcedx-vscode-apex/src/index.ts]: Cool, yes I'll send a follow up PR to clean up the naming.

Comment (lcampos) [reply packages/salesforcedx-vscode-apex/src/index.ts]: @AnanyaJha yes

**Pull request merged**